### PR TITLE
[FIX] board: Include noupdate data not detected by script

### PR DESCRIPTION
**Pull request by pedrobaeza**
The script doesn't inspect `<menuitem>` and other shortcut elements

@Tecnativa

### CODE REVIEW

**Review by rafaelbn — APPROVED** [review]
Tested in customer environment

File: addons/board/migrations/10.0.1.0/noupdate_changes.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+  <record model="ir.ui.menu" id="menu_board_my_dash">
```
Comment (MiquelRForgeFlow) [inline addons/board/migrations/10.0.1.0/noupdate_changes.xml:3]: I don't see what is the change. The exact same record is in v10. Nothing has changed.

Comment (pedrobaeza) [reply addons/board/migrations/10.0.1.0/noupdate_changes.xml:3]: Please check again: the parent is not the same.

Comment (MiquelRForgeFlow) [reply addons/board/migrations/10.0.1.0/noupdate_changes.xml:3]: v10: https://github.com/OCA/OCB/blob/10.0/addons/board/views/board_views.xml#L45

        parent="base.menu_board_root"
v11: https://github.com/OCA/OCB/blob/11.0/addons/board/views/board_views.xml#L45

        parent="base.menu_board_root"

Comment (pedrobaeza) [reply addons/board/migrations/10.0.1.0/noupdate_changes.xml:3]: You are looking the wrong branch :wink:

https://github.com/OCA/OCB/blob/9.0/addons/board/board_view.xml#L31

Comment (MiquelRForgeFlow) [reply addons/board/migrations/10.0.1.0/noupdate_changes.xml:3]: Ah, stupid of me. I saw your "added this to the **11.0** milestone 3 days ago" and that tricked me. :laughing:

Comment (pedrobaeza) [reply addons/board/migrations/10.0.1.0/noupdate_changes.xml:3]: Yeah, I changed it immediately because I clicked on the wrong line...

Comment (MiquelRForgeFlow) [reply addons/board/migrations/10.0.1.0/noupdate_changes.xml:3]: Hey, but the <code>id</code> is different! And also the <code>sequence</code>, <code>action</code> and <code>groups</code>.

Comment (pedrobaeza) [reply addons/board/migrations/10.0.1.0/noupdate_changes.xml:3]: Wrong link: https://github.com/OCA/OCB/blob/9.0/addons/board/board_mydashboard_view.xml#L45

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**